lib/components/dashboard/degree-info/distributionFunctions.tsx

```diff
@@ -7,7 +7,7 @@ export type requirements = {
   fulfilled_credits: number;
   description: string;
   exclusive?: boolean;
-  pathing?: boolean;
+  pathing?: number;
```

Comment:
If we have pathing as a number, can we now mix pathing with non-pathing requirements? ie. for a list of fine distributions, if pathing === 2, only the first two distributions are for pathing while the last few are mandatory.

Reply:
Pathing at the distributions level means that if the specific number of fine requirements within the distribution are satisfied, ONLY then is the distribution marked as complete

Reply:
I see gotcha